worker/src/main/proto/MahaCustomReportRequest.proto

```diff
@@ -0,0 +1,64 @@
+// Copyright 2017, Yahoo Holdings Inc.
+// Licensed under the terms of the Apache License 2.0. Please see LICENSE file in project root for terms.
+package com.yahoo.maha.worker.proto;
+
+option java_outer_classname = "MahaWorkerReportingProto";
+
+message MahaCustomReportRequest {
+    required bytes raw_request = 1;
+    required DeliveryMethod delivery_method = 2;
+    required QueryEngine query_engine = 3;
+    required OutputFormat output_format = 4;
+    required int64 job_id = 5;
+    required string schema = 6;
+    required ReportType report_type = 7;
+    optional string timezone = 8;
+    optional bool dryrun = 9;
+    optional int32 revision = 10;
+    optional int64 requestSubmittedTime = 11;
+    optional QueueType queueType = 12;
+    optional string userId = 13;
+    optional bool dualWrite = 14;
+    optional string registry_name = 15;
+}
+
+message MahaProcessedReport {
+    required int64 job_id = 1;
+    required DeliveryMethod delivery_method = 2;
+    required OutputFormat output_format = 3;
+    required bytes raw_request = 4;
+    required string schema = 5;
+    required string job_response = 6;
+    optional bool dryrun = 7;
+    optional QueueType queueType = 8;
+    optional string userId = 9;
+    optional bool dualWrite = 10;
+    optional string registry_name = 11;
+}
+
+enum DeliveryMethod {
+    EMAIL = 0;
+    MOBSTOR_AND_API = 1;
```

Comment:
Let's rename MOBSTOR_AND_API to CONTENT_STORE or something like that

Reply:
done